recipe/session/utils.go

```diff
@@ -57,6 +58,13 @@ func ValidateAndNormaliseUserInput(appInfo supertokens.NormalisedAppinfo, config
 		}
 	}
 
+	if config != nil && config.OlderCookieDomain != nil {
```

Comment:
how come we are not explicitly checking for it being equal to `""` like how we are doing in python? Does normaliseSessionScopeOrThrowError also work with `""` and return `""`?

Same comment goes for node sdk

Reply:
Yeah, the difference is how `url.Parse` works in go. It doesn't throw an error for empty string. I have added tests in all 3 SDKs for this case.